sopel/modules/dice.py

```diff
@@ -224,7 +224,7 @@ def _get_pretty_str(dice):
         result = eval_equation(eval_str)
     except TypeError:
         bot.reply("The type of this equation is, apparently, not a string. " +
```

Comment:
While you are at it, you can remove the `+` sign at the end, since Python strings are automatically concatenated with each other.

```suggestion
         bot.reply("The type of this equation is, apparently, not a string. "
```